nevergrad/optimization/oneshot.py

```diff
@@ -53,6 +59,10 @@ class RandomSearchMaker(base.ParametrizedFamily):
         Provides a random recommendation instead of the best point so far (for baseline)
     middle_point: bool
         enforces that the first suggested point (ask) is zero.
+    quasi_opposite: str
```

Comment:
not sure about the mode. Maybe `opposition_mode` ? And it would be better to have it `opposition_mode: Optional[str] = None` (instead of default being the "none" string). This makes it clearer that it is optional